R/utilities-units.R

```diff
@@ -15,7 +76,7 @@
 #'
 #' valuesInBaseUnit <- toBaseUnit(par, c(1000, 2000, 3000), "ml")
 #' @export
-toBaseUnit <- function(quantityOrDimension, values, unit, molWeight = NULL) {
+toBaseUnit <- function(quantityOrDimension, values, unit, molWeight = NULL, molWeightUnit = NULL) {

```

Comment:
To be consistent, both "toBaseUnit" and "toUnit" allow now molWeight in custom unit.